.github/workflows/main.yml

```
@@ -0,0 +1,41 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:  # yamllint disable-line rule:truthy
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Allow pushes/pr to all branches. Enough capacity to check everything.
```suggestion
on:  [push, pull_request, workflow_dispatch]
```